docs/pages/versions/unversioned/sdk/google-sign-in.md

```diff
@@ -37,16 +45,18 @@ For questions on setup, feel free to comment on this post: [**React Native Googl
       "bundleIdentifier": "example.expo.googlesignin",
       "config": {
         "googleSignIn": {
-          // Your REVERSE_CLIENT_ID from the GoogleService-info.plist
-          "reservedClientId": "<YOUR_IOS_CLIENT_ID>"
+          // Your REVERSED_CLIENT_ID from the GoogleService-Info.plist
```

Comment:
reSERVed or reVERSed?

Reply:
reserved. it's a particularly bad and entrenched typo.

https://github.com/expo/expo/pull/10274#issuecomment-696504888